src/conductor/client/http/api_client.py

```diff
@@ -198,6 +199,8 @@ def sanitize_for_serialization(self, obj):
                          for sub_obj in obj)
         elif isinstance(obj, (datetime.datetime, datetime.date)):
             return obj.isoformat()
+        elif isinstance(obj, uuid.UUID): # needed for compatibility with Python 3.7
```

Comment:
We've recently fixed a UUID serialization issue: https://github.com/conductor-sdk/conductor-python/pull/275/files but this won't work with Python 3.7 for example because in that Python version there won't be a TypeError (because UUID in Python 3.7 has `__dict__`). That's the reason for this if.